packs/agents-of-edgewatch-bestiary/shatterling.json

```diff
@@ -240,16 +240,11 @@
                 },
                 "rules": [
                     {
-                        "domain": "all",
-                        "key": "RollOption",
-                        "option": "blood-fueled-titter",
-                        "toggleable": true
-                    },
-                    {
+                        "hideIfDisabled": true,
                         "key": "FlatModifier",
                         "predicate": [
                             "action:demoralize",
-                            "blood-fueled-titter"
+                            "target:condition:persistent-damage:bleed"
```

Comment:
Not sure this works, since it's a "target within 30 feet of a creature with persistent bleed damage", not (necessarily) the target itself.